lightwood/mixers/helpers/default_net.py

```diff
@@ -46,21 +54,23 @@ def __init__(self, ds, dynamic_parameters):
             depth = 5
 
         # 3. Determine shpae based on the sizes & propotions
+        #if CONFIG.ALWAYS_OVERFIT:
+        #    shape = funnel(self.input_size,self.output_size,depth-2)
+        #    #shape = rombus(self.input_size,self.output_size,depth + 1,self.input_size*3)
+
         if (not large_input) and (not large_output):
-            if larger_input:
-                shape = rombus(self.input_size,self.output_size,depth,self.input_size*2)
-            else:
-                shape = rectangle(self.input_size,self.output_size,depth - 1)
+            shape = rombus(self.input_size,self.output_size,depth,self.input_size*2)
 
         elif (not large_output) and large_input:
             shape = funnel(self.input_size,self.output_size,depth)
 
         elif (not large_input) and large_output:
             shape = rectangle(self.input_size,self.output_size,depth - 1)
-
         else:
             shape = rectangle(self.input_size,self.output_size,depth - 2)
 
+        print(shape)
```

Comment:
print

Reply:
Was still debugging this branch and adding stuff, removed now